create3_lidar/config/mapper_params_online_async.yaml

```diff
@@ -0,0 +1,74 @@
+slam_toolbox:
```

Comment:
are these custom parameters?
if not, we should use the original config file from slam_toolbox

Reply:
There are some; I wonder if we can overlay them on top of the original config without duplicating the whole file?

Reply:
I don't recommend doing that (overlaying parameters can lead to hard to debug problems).
If we have custom parameters I prefer that we keep our config where we try to set everything that is needed